src/pages/Home.jsx

```diff
@@ -1,7 +1,328 @@
+import { useState, useEffect } from 'react';
+import {
+  Search,
+  ArrowRightCircle,
+  MapPin,
+  CalendarDaysIcon,
+  Star,
+  Scissors,
+  UsersIcon,
+  ShoppingBag,
+} from 'lucide-react';
+import image1 from '../assets/saloonImage1.jpg';
+import image2 from '../assets/saloonImage2.jpg';
+import image3 from '../assets/saloonImage3.webp';
+import gasabo from '../assets/gasabo.jpg';
+import nyarugenge from '../assets/nyarugenge.png';
+import kicukiro from '../assets/kicukiroDistrict.webp';
+import testimonial from '../assets/testimonial.jpg';
+import SearchInputs from '../components/SearchInputs';
+
 const Home = () => {
+  const [isHoverd, setIsHoverd] = useState(false);
+  const [current, setCurrent] = useState(0);
```

Comment:
The name can be improved here, `current`? current what? your names should be like`currentNumber`, `currentDistrict`